cmd/clusterctl/client/cluster/objectgraph.go

```diff
@@ -423,6 +424,7 @@ func (o *objectGraph) setSoftOwnership() {
 		// If the secret name is not a valid cluster secret name, ignore it.
 		secretClusterName, _, err := secretutil.ParseSecretName(secret.identity.Name)
 		if err != nil {
+			log.V(8).Info("ignore secret due to not fit purpose", "name", secret.identity.Name)
```

Comment:
What about?
```suggestion
			log.V(5).Info("Escluding secret from move (not linked with any Cluster)", "name", secret.identity.Name)
```
NB. Suggesting to use level 5 because in clusterctl logging standard there is _"Use logging Levels 5 for for providing all the information required for debug purposes/problem investigation."_

Reply:
~~Escluding~~ Excluding

Reply:
updated just now :) thanks~